_posts/2020-01-08-cockpit-210.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Cockpit 210 and Cockpit-podman 12
+author: kkoukiou
+date: 2020-01-08
+tags: cockpit linux podman
+slug: cockpit-210
+category: release
+summary: Cockpit with Overview, Dashboard and Machines improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 210.
+
+### Overview: Add CPU utilization to usage card
+
+Display CPU usage information in the usage card of the Overview page. For detailed CPU usage information, please consult the graphs page.
+
+![Overview](/images/overview-cpu-usage.png)
+
+### Dashboard: Support SSH identity unlocking when adding new machines
+
+Cockpit now offers an inline feature to unlock SSH identities when adding a new machine to the Dashboard.
+
+![unlock-keyring](/images/unlock-keyring.png)
+
+### SElinux: Introduce an Ansible automation script
+
+SELinux modifications can reviewed and exported as Ansible playbooks. These exported playbooks can be applied to other servers.
```

Comment:
Doesn't this miss a verb? "SELinux modifications can **be** reviewed.."
Also maybe I would change "...and exported as Ansible playbooks" to "and exported **also** as Ansible playbooks", since this change introduces another method (alongside existing shell script). Lastly, it is not "playbook", only "tasks".

Reply:
Yep. I was under a time crunch and working late yesterday due to GitHub throwing away half of my modifications (that are apparently not tracked in git anywhere). I had to rewrite things again and I guess I made a mistake. I hope it was fixed (haven't checked yet).